pkg/validation/options.go

```diff
@@ -353,6 +353,12 @@ func parseSignatureKey(o *options.Options, msgs []string) []string {
 		return msgs
 	}
 
+	// `logger` isn't configured yet
+	_, err := fmt.Fprintln(os.Stderr, "WARNING: `--signature-key` is deprecated")
```

Comment:
I'd be tempted to extend this and give a bit more context, WDYT?
```suggestion
	_, err := fmt.Fprintln(os.Stderr, "WARNING: `--signature-key` is deprecated. It will be removed in a future release")
```